config/core/webhooks/defaulting.yaml

```diff
@@ -28,3 +28,32 @@ webhooks:
   sideEffects: None
   name: webhook.serving.knative.dev
   timeoutSeconds: 10
+  rules:
+  - apiGroups:
+    - "*.knative.dev"
```

Comment:
This would capture eventing - even though the resource names might not match I wouldn't want a conflict so can we just list the three API groups we have in serving here?